torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -18,6 +19,8 @@
 from pathlib import Path
 
 class Model:
+    domain = Domain.NLP
+    task = Task.TRANSLATION
```

Comment:
BERT should be a language modeling task, right? transformers (and RNN/LSTM) could be used for different tasks but i'm pretty sure this BERT is configured for LM.

Reply:
OK, yeah agreed I'll fix this.